google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/settings/ImportSettings.java

```diff
@@ -1,185 +0,0 @@
-/*
- * Copyright (C) 2016 The Android Open Source Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.google.cloud.tools.intellij.settings;
-
-import com.google.cloud.tools.intellij.util.GctBundle;
-
-import com.intellij.ide.IdeBundle;
-import com.intellij.ide.actions.ImportSettingsFilenameFilter;
-import com.intellij.ide.plugins.PluginManager;
-import com.intellij.ide.startup.StartupActionScriptManager;
-import com.intellij.openapi.application.ApplicationManager;
-import com.intellij.openapi.application.ApplicationNamesInfo;
-import com.intellij.openapi.application.PathManager;
-import com.intellij.openapi.application.ex.ApplicationEx;
-import com.intellij.openapi.components.ExportableApplicationComponent;
-import com.intellij.openapi.components.ExportableComponent;
-import com.intellij.openapi.components.ServiceBean;
-import com.intellij.openapi.ui.Messages;
-import com.intellij.openapi.updateSettings.impl.UpdateSettings;
-import com.intellij.openapi.util.io.FileUtil;
-import com.intellij.util.io.ZipUtil;
-
-import java.io.File;
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
-import java.util.zip.ZipEntry;
-import java.util.zip.ZipException;
-import java.util.zip.ZipFile;
-
-/**
- * Provides functionality to update IDEA setting from a jar containing new settings.
- */
-public class ImportSettings {
```

Comment:
I noticed this code is totally unused and decided to delete it as part of this cleanup.